UI/build-extension.ts

```diff
@@ -12,7 +12,9 @@ const exists = async (filename: string): Promise<boolean> => {
         }
     }
 };
-const folderName = "build-extension";
+
+// https://stackoverflow.com/questions/3931741/why-does-make-think-the-target-is-up-to-date
+const folderName = ".build-extension";
```

Comment:
https://github.com/BlowaterNostr/blowater/commit/56c22123ed2a340473612779e753f52ab5d7a193 is the correct fix, you searched the wrong answer